build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.9.+'
+        classpath 'com.android.tools.build:gradle:1.2.3'
```

Comment:
Maybe i didn't do it right - but its here a9da3e62c9b9818e3a5a02adc238b67449bab033